tests/pytest.ini

```diff
@@ -7,11 +7,11 @@ addopts =
     -rs
     # capture only Python print and C++ py::print, but not C output (low-level Python errors)
     --capture=sys
-    # enable all warnings
-    -Wa
 filterwarnings =
     # make warnings into errors but ignore certain third-party extension issues
     error
+    # somehow, some DeprecationWarnings do not get turned into errors
+    always::DeprecationWarning
```

Comment:
Why, oh why, does python try so hard to hide deprecation warnings?!

Reply:
I have absolutely no clue. It gets funnier, btw: I added `warnings.warn("abc", DeprecationWarning)` and that _did_ get turned into an error!

But I didn't feel like wasting even more time, and digging further than https://github.com/python/cpython/blob/3c8d6934436e20163be802f5239c5b4e4925eeec/Objects/longobject.c#L226

Reply:
FYI, FutureWarning _is_ (as clarified by [PEP 565](https://www.python.org/dev/peps/pep-0565/#additional-use-case-for-futurewarning) the "user facing" deprecation warning.